packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -154,6 +154,12 @@ class BottomNavigationBar extends StatefulWidget {
   /// [selectedIconTheme] and [unselectedIconTheme], and both
   /// [IconThemeData.color] and [IconThemeData.size] must be set.
   ///
+  /// If set, [IconThemeData]s take precedence over [iconSize],
+  /// [selectedItemColor] and [unselectedItemColor].
+  ///
+  /// If set, [IconThemeData]s take precedence over [selectedItemColor] and [unselectedItemColor],
+  /// for the icons only
```

Comment:
This might be clearer as:

```suggestion
  /// [IconThemeData]s take precedence over [iconSize], [selectedItemColor]
  /// and [unselectedItemColor] for icons.
```

...but I'm not quite sure what "if set" means so it's hard for me to say (I'm not super familiar with this class). Does it specifically refer to selectedIconTheme/unselectedIconTheme, or is it referring to ambient themes?

Reply:
@Hixie It refers to selectedIconTheme/unselectedIconTheme. But you are correct in your assumed suggestion.